### PR TITLE
Create CODE_OF_CONDUCT.md

### LINKED ISSUE #18: Add a code of conduct for our project
**Is your feature request related to a problem? Please describe.**
As a contributor, I want a code of conduct so that we have guidelines for developers to follow.
**Describe the solution you'd like**
A Code of Conduct to be placed in the repository that outlines our community guidelines.
**Describe alternatives you've considered**
N/a
**Additional context**
N/a

### PR DESCRIPTION
**Pull request by ParieMalhotra**
Adding in a code of conduct from a template

Fixes #18 

## Proposed Changes

  - Addition of Code of Conduct to the project.
  - Enforcement of CoC section addresses course co-ordinator

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at k.blincoe@auckland.ac.nz. All
```
Comment (ErnWong) [inline CODE_OF_CONDUCT.md]: Did Kelly tell us to not put her email online - and instead just mention "course supervisor"?

**Comment by ErnWong** [comment]
Otherwise, the code of conduct looks good!

It would be really awesome if you could also edit your PR description to fill in the corresponding issue number and the proposed changes bullet points. (See other PRs for inspiration)

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (ParieMalhotra) [reply CODE_OF_CONDUCT.md]: Yup she did! I forgot about that but thanks for the reminder. It said to enter an email so I didn't think twice :)

**Comment by ParieMalhotra** [comment]
> Otherwise, the code of conduct looks good!
> 
> It would be really awesome if you could also edit your PR description to fill in the corresponding issue number and the proposed changes bullet points. (See other PRs for inspiration)

Right yup, I have looked at other PRs and now know what you mean. Not sure how to do it though as I can't seem to find where to edit it.

**Comment by ParieMalhotra** [comment]
> > Otherwise, the code of conduct looks good!
> > It would be really awesome if you could also edit your PR description to fill in the corresponding issue number and the proposed changes bullet points. (See other PRs for inspiration)
> 
> Right yup, I have looked at other PRs and now know what you mean. Not sure how to do it though as I can't seem to find where to edit it.

Nevermind, I found it and have now updated it.

**Pull request merged**